app/build.gradle

```diff
@@ -1,5 +1,12 @@
 apply plugin: 'com.android.application'
 
+allprojects {
+    repositories {
+        jcenter()
+        maven { url 'https://maven.google.com' }
```

Comment:
Unfortunately, Room persistence (and all of android architecture component) is only available from that repo at the moment. 

As you have noticed [here](https://gitlab.com/fdroid/fdroidserver/merge_requests/275), changes required to use maven.google.com on f-droid is already merged, and waiting to become available on the server.  

I say we wait for a bit to see what happens.